### ISSUE TITLE
Installation failing no matter what tried

### ISSUE THREAD

**Issue by FusionFF** [thecodingmachine/workadventure#1209]
Hello,
I need to install Workadventure on a Raspberry.
I got the latest stable, v1.4.0 and tried to install it, but nothing worked.
I tried the method indicated with README.md for a working developpment environment, but all I get in the end when browsing the app is a "404 not found".

I tried installing it on a Debian Buster VM on my work computer to see if the issue comes from Raspberry, this time when browsing I get a "Bad gateway" error.

I tried running the docker-compose up -d as root rather than userland to see if it's the problem, nope, same error.

I also tried using the "production ready" docker-compose file provided, but the build failed on an error of yarn getting a permission denied on /var/www/messages/node_modules, even when running it as root...

So, basically I don't know what to do, nothing I try work, and seeing that the README guide is basically black magic as "just run that command it will work", debug isn't easy. Don't you have a manual install guide or something like that? It's usually a lot more reliable than docker "not so reproductible" environments.

**Comment by jonnytest1** [comment]
for docker you need to build your own images since the normal images arent available for the docker architecture

and well ... for the vm you probably just screwed up in the network config somewhere (maybe missing port forwarding into the vm ?)

**Comment by FusionFF** [comment]
For Docker, I'll check that part for the Raspberry, thanks.
For the VM, I tried with a desktop VM using purely localhost (installing all in the VM and browsing it locally), I wanted to check if I could at least get the "dev install" indicated in the README working, and it ends up in "Bad gateway" error (or a "404 not found" error sometimes, it seems to be either one or the other, randomly when cleanly reinstalling the VM). However, can't manage to get the "prod install" working no matter what, the build always fail on this "permission denied" error said above.

**Comment by moufmouf** [comment]
What operating system are you using (for the prod install?) Windows? MacOS? Linux?

**Comment by bujinin** [comment]
I had the same problem with the "permission denied error". Fixed it by adding "USER root" to the dockerfile in line 4, so it looks like this:
```                           
FROM thecodingmachine/workadventure-back-base:latest as builder
WORKDIR /var/www/messages
COPY --chown=docker:docker messages .
USER root
RUN yarn install && yarn proto

# we are rebuilding on each deploy to cope with the PUSHER_URL environment URL
FROM thecodingmachine/nodejs:14-apache

COPY --chown=docker:docker front .
COPY --from=builder --chown=docker:docker /var/www/messages/generated /var/www/>

# Removing the iframe.html file from the final image as this adds a XSS attack.
# iframe.html is only in dev mode to circumvent a limitation
RUN rm dist/iframe.html

RUN yarn install

ENV NODE_ENV=production
ENV STARTUP_COMMAND_0="./templater.sh"
ENV STARTUP_COMMAND_1="yarn run build"
ENV APACHE_DOCUMENT_ROOT=dist/
```
Hope that helps!

**Comment by moufmouf** [comment]
This was fixed a while ago I think.
Thanks for reporting the issue!
